exporters/prometheus/exporter.go

```diff
@@ -147,6 +151,14 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 		}
 
 		for _, m := range scopeMetrics.Metrics {
+			drop, m, err := c.validateMetrics(m)
+			if err != nil {
+				fmt.Printf("warning: %v\n", err)
```

Comment:
Also, if we drop a metric, we should ideally give the user instructions to fix it (e.g. use a view that selects for a  name + instrumentation library, and outputs a different name)